packages/fragment/tests/fragments/special-symbols-1.xml

```diff
@@ -1,7 +1,6 @@
 <FRAGMENT parents="p">
     <copyright />
     <phonomark/>
-    <copyleft />
```

Comment:
This one isn't in the schema but is in the sample article...Maybe we should file an issue to Rob to add it to the schema?

Reply:
See https://github.com/PreTeXtBook/pretext/pull/2020.